src/backend/api/meals.js

```diff
@@ -2,6 +2,42 @@ const express = require('express');
 const router = express.Router();
 const knex = require('../database');
 
+router.get('/', async (req, res) => {
+  try {
+    const result = await knex.raw(`SELECT * FROM meal`);
+    res.json({
+      meals: result[0],
+    });
+  } catch (error) {
+    res.status(500).json({ Msg: 'Internal server error' });
+  }
+});
+
+router.get('/future-meals', async (req, res) => {
+  try {
+    const result = await knex.raw(
+      `SELECT * FROM meal ORDER BY created_date DESC `
+    );
+    res.json({
+      meals: result[0],
+    });
+  } catch {
+    res.status(500).json({ Msg: 'Internal server error' });
+  }
+});
+
+router.get('/past-meals', async (req, res) => {
+  try {
+    const result = await knex.raw(
+      `SELECT * FROM meal ORDER BY created_date ASC`
+    );
+    res.json({
+      meals: result[0],
+    });
+  } catch {
+    res.status(500).json({ Msg: 'Internal server error' });
+  }
+=======
```

Comment:
@viedamayanti Seems like something went wrong with the merge. I don't think this file works now.

Reply:
@orhantoy  I am not sure what I did, what do you mean the file does not work? on my local? or my github? what should I do?

Reply:
I would expect this line to cause the JavaScript syntax to be invalid. Try to run it locally.